yt_dlp/extractor/afreecatv.py

```diff
@@ -482,3 +479,62 @@ def _real_extract(self, url):
             'formats': formats,
             'is_live': True,
         }
+
+
+class AfreecaTVUserIE(InfoExtractor):
+    IE_NAME = 'afreecatv:user'
+    _VALID_URL = r'https?://bj\.afreeca(?:tv)?\.com/(?P<id>[^/]+)/vods/?(?P<slug_type>[^/]+)?'
+    _TESTS = [{
+        'url': 'https://bj.afreecatv.com/ryuryu24/vods/review',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'ryuryu24',
+            'title': 'ryuryu24 - review',
+        },
+        'playlist_count': 218,
+    }, {
+        'url': 'https://bj.afreecatv.com/parang1995/vods/highlight',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'parang1995',
+            'title': 'parang1995 - highlight',
+        },
+        'playlist_count': 997,
+    }, {
+        'url': 'https://bj.afreecatv.com/ryuryu24/vods',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'ryuryu24',
+            'title': 'ryuryu24 - all',
+        },
+        'playlist_count': 221,
+    }, {
+        'url': 'https://bj.afreecatv.com/ryuryu24/vods/balloonclip',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': 'ryuryu24',
+            'title': 'ryuryu24 - balloonclip',
+        },
+        'playlist_count': 0,
+    }]
+
+    PER_PAGE = 60
+
+    def _real_extract(self, url):
+        user_id, slug_type = self._match_valid_url(url).group('id', 'slug_type')
+        user_type = slug_type if slug_type else "all"
+        entries, page = [], 1
+        while True:
+            info = self._download_json(
+                f'https://bjapi.afreecatv.com/api/{user_id}/vods/{user_type}',
+                video_id=user_id, query={'page': page, 'per_page': self.PER_PAGE, 'orderby': 'reg_date'},
+                note=f'Download [{user_type}] video page {page} of {user_id}')
+            for item in info.get('data'):
+                if item:
+                    entries.append(self.url_result(f'https://vod.afreecatv.com/player/{item.get("title_no")}/',
+                                                   AfreecaTVIE.ie_key(), item.get("title_no")))
+            if not try_get(info, lambda x: x['links']['next']):
+                break
+            page += 1
```

Comment:
create a paged list

Reply:
Can you explain to me more specifically what you want?

Reply:
See `LBRYChannelIE` for example of how to use a `PagedList`

Reply:
Thank you so much, let me check

Reply:
I write short comments as reviews so that I can get through these quickly; and since u r not a first-time contributor, I assume you'll understand it nonetheless. If any of my comments is not clear, always feel free to ask for clarification

Reply:
Sure, if has any question, i will ask again, ths you.